clients/drcachesim/tracer/output.cpp

```diff
@@ -659,7 +665,9 @@ create_v2p_buffer(per_thread_t *data)
 static bool
 is_ok_to_split_before(trace_type_t type)
 {
-    return type_is_instr(type) || type == TRACE_TYPE_INSTR_MAYBE_FETCH ||
+    return (op_instr_encodings.get_value()
+                ? type == TRACE_TYPE_ENCODING
```

Comment:
May be add a comment that this is based on which is the first entry output for each instrumentation sequence.

Reply:
Done.